tests/test_minimal/test_tools/test_backend_and_dataset_configuration/test_helpers/test_get_default_backend_configuration.py

```diff
@@ -262,3 +263,22 @@ def test_complex_zarr(zarr_nwbfile_path):
 
 """
     assert stdout.getvalue() == expected_print
+
+
+def test_000_ImageSeries():
+    nwbfile = mock_NWBFile()
+
+    im_series = ImageSeries(
+        name="my_video", external_file=["my_video.mp4"], starting_frame=[0], format="external", rate=30.0
+    )
+    nwbfile.add_acquisition(im_series)
+
+    with NWBHDF5IO("test.nwb", "w") as io:
+        io.write(nwbfile)
+
+    io = NWBHDF5IO("test.nwb", "r")
+    nwbfile = io.read()
+    print(nwbfile.acquisition["my_video"])
+
+    backend_config = get_default_backend_configuration(nwbfile, "hdf5")
+    print(backend_config)  # TODO: Figure out why this doesn't throw an error like Ben said it did
```

Comment:
Can we add an integration test where we write a new NWB file with a different backend configuration?

Reply:
Hmm... You mean something like 'make a Zarr-backend copy of this existing HDF5-backed NWB file'? Or same type of backend (such as both HDF5) but different configuration?

Reply:
good question. Both, I guess. I just want to make sure this works end-to-end and does not require us to do anything funky in practice